__tests__/validators/options_processor/options/or.test.js

```diff
@@ -1,7 +1,7 @@
 const or = require('../../../../lib/validators/options_processor/options/or')
 
 const validatorContext = {
-  name: 'label',
+  name: 'OptionName',
```

Comment:
Changed it to a generic name to clarify that value of the string does not matter in this fixture.